src/Picqer/Carriers/SendCloud/Connection.php

```diff
@@ -112,12 +112,16 @@ public function apiUrl()
      * Perform a GET request
      * @param string $url
      * @param array $params
+     * @param string $auth
      * @return array
      * @throws SendCloudApiException
      */
-    public function get($url, $params = [])
+    public function get($url, $params = [], $auth = false)
     {
         try {
+            if ($auth) {
+                $params[$auth] = $this->apiKey;
```

Comment:
Shouldn't this be `$params['auth']` instead? Now it'll expand to `$params[true]`